### DISCUSSION TITLE
container/linux-vnet

### DISCUSSION THREAD

**Discussion by giscus** [isno/theByteBook#145] [General]
# container/linux-vnet

构建大规模高可用的分布式系统

https://www.thebyte.com.cn/container/linux-vnet.html

**Comment by dentonliu** [comment]
IETF 定义了 XXLAN 规范   应该是  VXLAN

**Reply by isno** [reply]
谢谢纠正，我写错了。

**Comment by xdsdmg** [comment]
“一方面通过 VXLAN 中的 24 比特 VNI 字段（如图 1-5 所示）”应该是如图 2-27

**Reply by isno** [reply]
谢谢，已修正。
